src/ci/docker/host-x86_64/dist-x86_64-linux/Dockerfile

```diff
@@ -82,6 +82,12 @@ COPY host-x86_64/dist-x86_64-linux/build-clang.sh /tmp/
 RUN ./build-clang.sh
 ENV CC=clang CXX=clang++
 
+ENV PERF_COMMIT 1e19fc4c6168d2f7596e512f42f358f245d8f09d
```

Comment:
Hmm, what's the strategy for updating this? Presumably `collector` could change in the future.

Oh, I just saw this was hard-coded before, so this isn't changing anything.

Reply:
Yeah, this isn't really critical to update on any regular cadence. I don't expect us to update the hash here unless the benchmarks we're using break in rustc updates (some of them use unstable features), but that is rare and should be relatively easy to work around if necessary.